cli/src/commands/branch.rs

```diff
@@ -100,9 +100,13 @@ pub struct BranchDeleteArgs {
 pub struct BranchListArgs {
     /// Show all tracking and non-tracking remote branches including the ones
     /// whose targets are synchronized with the local branches.
-    #[arg(long, short, conflicts_with_all = ["names", "revisions"])]
+    #[arg(long, short, conflicts_with_all = ["names", "revisions", "tracked"])]
     all: bool,
 
+    /// Show remote tracked branches only. Omits the git remote by default.
```

Comment:
(here, and in docs) "the git remote" sounds like a remote repository of git backend (i.e. normal remotes.)

We use "Git-tracking branches" in several places instead. I don't know if it's clear enough, but I think it's less confusing.

Reply:
What about "Omits local git-tracking branches by default"?

Reply:
> Omits local git-tracking branches by default

Much better. Let's use capitalized "Git-tracking" for consistency.

Reply:
Done!

Reply:
I resolved the discussions since you approved, hopefully that's fine!

Reply:
Sure, feel free to merge it.

Reply:
I don't think I have write access to merge it

Reply:
> I don't think I have write access to merge it

^ @martinvonz ?

Reply:
Sent an invite. Thanks for the PR! And thanks to Yuya for reviewing!